lib/normalizer.c

```diff
@@ -1892,6 +1892,476 @@ grn_nfkc_normalize_unify_katakana_trailing_o(grn_ctx *ctx,
   return current;
 }
 
+static const unsigned char *
+grn_nfkc_normalize_unify_kana_prolonged_sound_mark(grn_ctx *ctx,
+                                                   const unsigned char *start,
+                                                   const unsigned char *current,
+                                                   const unsigned char *end,
+                                                   size_t *n_used_bytes,
+                                                   size_t *n_used_characters,
+                                                   unsigned char *unified_buffer,
+                                                   size_t *n_unified_bytes,
+                                                   size_t *n_unified_characters,
+                                                   void *user_data)
+{
+  size_t char_length;
+  const unsigned char *previous;
+  size_t *previous_length = user_data;
+
+  char_length = (size_t)grn_charlen_(ctx, current, end, GRN_ENC_UTF8);
+
+  *n_used_bytes = char_length;
+  *n_used_characters = 1;
+
+  if (*previous_length == 3 &&
+      grn_nfkc_normalize_is_prolonged_sound_mark_famity(current, char_length)) {
```

Comment:
Really?
I think that we should use only `U+30FC KATAKANA-HIRAGANA PROLONGED SOUND MARK` here.
If an user wants normalized prolonged sound mark, the user should use `unify_prolonged_sound_mark`.

Reply:
OK, I will fix it.

Reply:
Also I will add a test use this option with `unify_prolonged_sound_mark`.

Reply:
Fixed.